src/styles/blocks/recomended.css

```diff
@@ -0,0 +1,43 @@
+.recomended__item {
+  margin-bottom: 40px;
```

Comment:
Styles should be written in the `main.scss` file according to the task description.